cmake/therock_subproject.cmake

```diff
@@ -50,6 +50,12 @@ set(THEROCK_AMD_LLVM_DEFAULT_CXX_FLAGS
   -Wno-unused-command-line-argument
 )
 
+if (WIN32)
```

Comment:
Style nit: this project consistently doesn't put a space between if and the paren.